framework/utils/InputData.py

```diff
@@ -277,7 +277,7 @@ def addSub(cls, sub, quantity=Quantity.zero_to_infinity):
     subsSet = cls._subDict.get(sub.getName(), set())
     if (len(subsSet) == 1 and next(iter(subsSet))._checkCanRead is None) or \
        (len(subsSet) > 0 and sub._checkCanRead is not None):
-       print("ERROR adding checked and unchecked to", sub.getName()," in ",
+       print("INPUT SPECWARNING adding checked and unchecked to", sub.getName()," in ",
```

Comment:
I think we should comment this out and put it just within a debug print:

```
if __debug__:
  print("INPUT SPECWARNING adding checked and unchecked to", sub.getName()," in ",
```
so we activate it only when we turn on the assert...

Reply:
I know it is not really part of the PR, but that print is very annoying in production calculations :D

Reply:
agreed! Done.